packages/testcontainers/src/MetaChainContainer.ts

```diff
@@ -36,13 +36,19 @@ export class MetaChainContainer extends GenericContainer {
     ];
   }
 
-  public async start(): Promise<StartedMetaChainContainer> {
+  public async start(configModifiers = {}): Promise<StartedMetaChainContainer> {
```

Comment:
Take a look at this, it should be structured similarly to this:

Reply:
Read all the reviews here: https://github.com/JellyfishSDK/jellyfish/pull/1790#pullrequestreview-1128780085

Reply:
Mainly this https://github.com/JellyfishSDK/jellyfish/pull/1790#discussion_r986069192